packages/core/src/util.ts

```diff
@@ -238,7 +232,7 @@ export class Util {
             break;
         case TestExecutionMode.Combined:    
             for (const locator of config.locators) {
-                const record = locatorMap.get(locator.numberofexecutions) ?? [];
+                let record = locatorMap.get(locator.numberofexecutions) ?? [];
```

Comment:
why `let`? Also, can you run `yarn lint` on your branch? [Will add as CI step later on]